org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/extensions/contentmodel/XMLSchemaCompletionExtensionsTest.java

```diff
@@ -301,6 +301,33 @@ public void completionWithoutStartBracket() throws BadLocationException {
 		
 	}
 
+	@Test
+

```

Comment:
Please add test with xmlns:MyXsi=\"http://www.w3.org/2001/XMLSchema-instance\"\ to check you have completion with MyXsi:nil, etc